src/Logger/BanditActionEvent.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Eppo\Logger;
+
+use DateTime;
+use Eppo\DTO\Bandit\Bandit;
+use Eppo\DTO\Bandit\BanditEvaluation;
+use JsonSerializable;
+use Serializable;
+
+class BanditActionEvent implements Serializable, JsonSerializable
```

Comment:
Why does this need to be JsonSerializable? User-friendliness?

Reply:
A number of the telemetry libraries are able to take data objects that are JSONSerializable in whatever language they're implemented in. Makes implementations of `IBanditLogger` pretty straightforward.

Reply:
Ah right, logger-friendly 👍